pennylane/qchem/openfermion_obs.py

```diff
@@ -909,10 +924,11 @@ def molecular_hamiltonian(
         geometry_dhf = qml.numpy.array(coordinates)
         geometry_hf = coordinates.flatten()
 
+    if wires:
```

Comment:
If we define `wire_map` as `None` here then we would need just one `if-else` block at line `990`.